docker/datahub-ingestion/base-requirements.txt

```diff
@@ -0,0 +1,313 @@
+absl-py==1.1.0
```

Comment:
How did you geneated this file?
What should be the process to regenerate this file if somebody adds/pins a new/existing dep in ingestion?

Reply:
I added a pip freeze in https://github.com/datahub-project/datahub/pull/5225 to get list of dependencies. If the build time starts to become large again then we can take the freeze from that and update this file.